ui/js/component/modalIncompatibleDaemon/index.js

```diff
@@ -1,13 +1,12 @@
 import React from "react";
 import { connect } from "react-redux";
 import { doQuit, doSkipWrongDaemonNotice } from "actions/app";
```

Comment:
Is this `doSkipWrongDaemonNotice` required to be here? I could not find the corresponding command in `actions/app`.